### PR TITLE
Protect: Add formatting help text to IP list controls

### PR DESCRIPTION
**Pull request by nateweller**
**Merge after https://github.com/Automattic/jetpack/pull/39425 to ensure accuracy of information.**

## Proposed changes:

* Adds a help text snippet below the IP list text inputs, describing the supported formatting.

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [x] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion

Related to https://github.com/Automattic/jetpack/pull/39425

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

* Review the UI and confirm it matches the screenshot below

## Screenshots

Screenshot 2024-09-18 at 3 17 36 PM

### CODE REVIEW

File: projects/plugins/protect/src/js/routes/firewall/index.jsx [hunk 1]
```diff
@@ -468,6 +468,14 @@ const FirewallPage = () => {
 							onChange={ handleChange }
 							disabled={ ! canEditFirewallSettings || ! ipBlockListEnabled }
 						/>
+						{ ipBlockListEnabled && (
+							<Text variant="body-extra-small" mt={ 1 }>
+								{ __(
+									'IPv4 and IPv6 supported. Separate IPs with commas, spaces, or new lines. To specify a range, use CIDR notation (i.e. 12.12.12.0/24) or enter the low value and high value separated by a dash (i.e.12.12.12.0–12.12.12.255).',
```
Comment (ArSn) [inline projects/plugins/protect/src/js/routes/firewall/index.jsx]:
```suggestion
									'IPv4 and IPv6 supported. Separate IPs with commas, spaces, or new lines. To specify a range, use CIDR notation (i.e. 12.12.12.0/24) or enter the low value and high value separated by a dash (i.e. 12.12.12.0–12.12.12.255).',
```

File: projects/plugins/protect/src/js/routes/firewall/index.jsx [hunk 2]
```diff
@@ -516,12 +524,14 @@ const FirewallPage = () => {
...
-							<Text variant="body-extra-small" mt={ 1 }>
-								{ __(
-									"Add IP addresses here to ensure they always have access to your site, regardless of Jetpack's security features.",
-									'jetpack-protect'
-								) }
-							</Text>
+							{ jetpackWafIpAllowListEnabled && (
+								<Text variant="body-extra-small" mt={ 1 }>
+									{ __(
+										'IPv4 and IPv6 supported. Separate IPs with commas, spaces, or new lines. To specify a range, use CIDR notation (i.e. 12.12.12.0/24) or enter the low value and high value separated by a dash (i.e.12.12.12.0–12.12.12.255).',
```
Comment (ArSn) [inline projects/plugins/protect/src/js/routes/firewall/index.jsx]:
```suggestion
										'IPv4 and IPv6 supported. Separate IPs with commas, spaces, or new lines. To specify a range, use CIDR notation (i.e. 12.12.12.0/24) or enter the low value and high value separated by a dash (i.e. 12.12.12.0–12.12.12.255).',
```

**Review by ArSn — CHANGES_REQUESTED** [review]
Small typo there, rest looks good.

**Comment by nateweller** [comment]
@ArSn Thanks, applied the typo fix :+1:

**Review by ArSn — APPROVED** [review]
Almost missed this. LGTM. 👍🏻

**Pull request merged**